### ISSUE TITLE
Define Configurator GUI Grid

### ISSUE THREAD

**Issue by briankyncl** [briankyncl/aboutthiscomputer#14]
## Background
After GUI has been wireframed, the wireframe needs to be converted into a grid of variables defining overall columns and rows, sub-columns and sub-rows, element sizing and spacing, etc.

## Desired Outcome
With the GUI grid fully defined, building the GUI should be as simple as "filling in the blanks"; iterating through the grid and assigning GUI elements to the positions defined by the grid.

## TL;DR
Convert wireframe into grid.
